cryptotik/hitbtc.py

```diff
@@ -0,0 +1,71 @@
+import requests
+from .common import APIError, headers
+
+class Hitbtc:
+
+    url = 'http://api.hitbtc.com/api/1/public/'
+    delimiter = ""
+    headers = headers
+
+    @classmethod
+    def format_pair(cls, pair):
+        """format the pair argument to format understood by remote API."""
+        pair = pair.replace("-", cls.delimiter).upper()
+        return pair
+
+    @classmethod
+    def api(cls, url):
+        '''call api'''
+
+        try:
+            result = requests.get(url, headers=cls.headers, timeout=3)
+            assert result.status_code == 200
+            return result.json()
+        except requests.exceptions.RequestException as e:
+            raise APIError(e)
+
+    @classmethod
+    def get_market_ticker(cls, pair):
+        '''returns simple current market status report'''
+
+        return cls.api(cls.url + cls.format_pair(pair) + "/ticker")
+
+    @classmethod
+    def get_market_trade_history(cls, pair, since=None):
```

Comment:
Do you need since here? It seems that you do not use any form of time parameter in the function bellow.